spec/support/assert_select_root.rb

```diff
@@ -0,0 +1,9 @@
+module AssertSelectRoot
+  def document_root_element
+    html_document.root
+  end
+end
+
+RSpec.configure do |config|
+  config.include AssertSelectRoot, :type => :request
+end
```

Comment:
I've never seen it before, why'd we need to do it?  (I'm 100% sure we need to just wondering)